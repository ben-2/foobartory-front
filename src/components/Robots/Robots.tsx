import styles from './Robot.module.css';
import { Card } from '@material-ui/core';

export function Robots() {
  return (
    <>
      <Card>
        Robots
      </Card>
    </>
  );
}
