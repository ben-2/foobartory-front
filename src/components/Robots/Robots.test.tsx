import { Robots } from './Robots';
import { shallow } from 'enzyme';

describe('<Robots />', () => {
  it('renders an emtpy Paper Card when the game starts', () => {
    const wrapper = shallow(<Robots />)
    expect(wrapper).toMatchSnapshot();
  })
});
