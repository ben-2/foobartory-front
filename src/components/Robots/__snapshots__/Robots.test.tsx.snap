// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Robots /> renders an emtpy Paper Card when the game starts 1`] = `ShallowWrapper {}`;
